src/commands/notify-on-failure.yml

```diff
@@ -0,0 +1,78 @@
+description: |
+  Send a notification with Pushover on failure
+
+# NOTE: from the Message API (https://pushover.net/api), I'm only exposing
+# the interface I need at the moment, which means:
+#
+# - title is required
+# - many options aren't here
+#
+# PRs welcome
+parameters:
+  api-key:
+    description: Pushover API Key
+    type: env_var_name
+    default: PUSHOVER_API_KEY
+  user-key:
+    description: Pushover User Key
+    type: env_var_name
+    default: PUSHOVER_USER_KEY
+  title:
+    description: Notification title
+    type: string
+    default: "Failed ${CIRCLE_PROJECT_REPONAME}'s job (${CIRCLE_JOB})"
+  message:
+    description: Notification message
+    type: string
+    default: "Failed ${CIRCLE_USERNAME:=unknown}'s build (#${CIRCLE_BUILD_NUM}) in ${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME} (${CIRCLE_BRANCH})"
+  priority:
+    description: "Notification priority (low-to-high: -2 to 1)"
+    type: integer
+    default: 1
+  only_for_branches:
+    description: If set, a space-separated list of branches for which to send notifications.
+    type: string
+    default: master
+
+steps:
+  - run:
+      when: on_success
+      name: Pushover - Setting Success Condition
+      command: |
+        echo 'export PUSHOVER_BUILD_STATUS="success"' > /tmp/PUSHOVER_JOB_STATUS
```

Comment:
This is an idea of Slack orb. I can remove the dependency with /bin/bash (BASH_ENV).
https://github.com/CircleCI-Public/slack-orb/blob/v4.2.0/src/commands/notify.yml#L61